### PR TITLE
update: add what's new in Kotlin 1.4.30

### PR DESCRIPTION
**Pull request by p7nov**

### CODE REVIEW

File: pages/docs/reference/whatsnew1430.md [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+
+[**Standard library**](#standard-library)
+* [Locale-agnostic API for upper/lowercasing text](#locale-agnostic-api-for-upperlowercasing-text)
+* [Clear Char-to-code and Char-to-digit conversions](#clear-char-to-code-and-char-to-digit-conversions)
+
+[**Serialization updates**](#serialization-updates)
+- [Inline classes serialization support](#inline-classes-serialization-support)
+- [Unsigned primitive type serialization support](#unsigned-primitive-type-serialization-support)
+
+## Language features
```
Comment (katia-energizer) [inline pages/docs/reference/whatsnew1430.md:40]: Do we want to add a link to the language features blog post?

File: pages/docs/reference/whatsnew1430.md [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+
+  Starting with 1.4.30, the Kotlin compiler uses a new mangling scheme by default. Use the `-Xuse-14-inline-classes-mangling-scheme`
+  compiler flag to force the compiler to use the old 1.4.0 mangling scheme and preserve binary compatibility.
+
+Kotlin 1.4.30 promotes inline classes to Beta, and we are planning to make them Stable in future releases. We’d be very
+grateful if you would share your feedback with us using this [YouTrack ticket](https://youtrack.jetbrains.com/issue/KT-42434).
+
+To try the preview version of inline classes, add the compiler option `-Xinline-classes` or `-language-version 1.5`.
+
+Learn more about the mangling algorithm in [KEEP](https://github.com/Kotlin/KEEP/blob/master/proposals/inline-classes.md).
```
Comment (katia-energizer) [inline pages/docs/reference/whatsnew1430.md:222]: Link to updated docs?

File: pages/docs/reference/whatsnew1430.md [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+   override fun fill() { /*...*/ }
+}
+```
+</div>
+
+To try the preview version of sealed interfaces, add the compiler option `-language-version 1.5`. Once you switch to this
+version, you’ll be able to use the `sealed` modifier on interfaces. We’d be very grateful if you would share your feedback
+with us using this [YouTrack ticket](https://youtrack.jetbrains.com/issue/KT-42433).
+
+[Learn more about sealed interfaces](sealed-classes.html).
```
Comment (katia-energizer) [inline pages/docs/reference/whatsnew1430.md:133]: Don't you want to add a link to a specific section about sealed interfaces?

File: pages/docs/reference/whatsnew1430.md [hunk 4]
```diff
@@ -0,0 +1,473 @@
...
+Sealed classes can now form more flexible hierarchies. They can have subclasses in all files of the same compilation unit
+and the same package. Previously, all subclasses had to appear in the same file.
+
+Direct subclasses may be top-level or nested inside any number of other named classes, named interfaces, or named objects.
+The subclasses of a sealed class must have a name that is properly qualified – they cannot be local nor anonymous objects.
+
+To try package-wide hierarchies of sealed classes, add the compiler option `-language-version 1.5`.
+We’d be very grateful if you would share your feedback with us using this [YouTrack ticket](https://youtrack.jetbrains.com/issue/KT-42433).
+
+[Learn more about package-wide hierarchies of sealed classes](sealed-classes.html).
```
Comment (katia-energizer) [inline pages/docs/reference/whatsnew1430.md:150]: Don't you want to add a link to a specific section about sealed interfaces?

File: pages/docs/reference/whatsnew1430.md [hunk 5]
```diff
@@ -0,0 +1,473 @@
...
+Kotlin 1.4.30 promotes inline classes to Beta, and we are planning to make them Stable in future releases. We’d be very
+grateful if you would share your feedback with us using this [YouTrack ticket](https://youtrack.jetbrains.com/issue/KT-42434).
+
+To try the preview version of inline classes, add the compiler option `-Xinline-classes` or `-language-version 1.5`.
+
+Learn more about the mangling algorithm in [KEEP](https://github.com/Kotlin/KEEP/blob/master/proposals/inline-classes.md).
+
+## Kotlin/JVM
+
+### JVM IR compiler backend reaches Beta
```
Comment (katia-energizer) [inline pages/docs/reference/whatsnew1430.md:226]: Do we want to add a link to a dedicated blog post?

File: pages/docs/reference/whatsnew1430.md [hunk 6]
```diff
@@ -0,0 +1,473 @@
...
+```kotlin
+fun Int.digitToChar(radix: Int): Char
+```
+</div>
+
+See more details in [KEEP](https://github.com/Kotlin/KEEP/blob/master/proposals/stdlib/char-int-conversions.md).
+
+## Serialization updates
+
+Along with Kotlin 1.4.30, we are releasing `kotlinx.serialization` [1.1.0](https://github.com/Kotlin/kotlinx.serialization/releases/tag/v1.1.0),
```
Comment (katia-energizer) [inline pages/docs/reference/whatsnew1430.md:424]: Do we release 1.1.0 with 1.4.30? @qwwdfsad wrote about 1.1.0-RC a few days ago.

File: pages/docs/reference/whatsnew1430.md [hunk 7]
```diff
@@ -0,0 +1,473 @@
+---
+type: doc
+layout: reference
+title: "What's New in Kotlin 1.4.30"
+---
+
+# What's New in Kotlin 1.4.30
+
+Kotlin 1.4.30 offers preview versions of new language features, promotes the new IR backend of the Kotlin/JVM compiler to
```
Comment (katia-energizer) [inline pages/docs/reference/whatsnew1430.md:9]: Do we want to add a link to the release blog post? I would add.

**Review by katia-energizer — APPROVED** [review]
LGTM 👍. My comments are about links. I would add links to release and specific blog posts, and links to docs where they are available.

File: pages/docs/reference/whatsnew1430.md (hunk 3 above)
Comment (p7nov) [reply pages/docs/reference/whatsnew1430.md:133]: Yeah, sure. I just couldn't do it before the doc update is merged. And the same for the link in the next section.

File: pages/docs/reference/whatsnew1430.md (hunk 6 above)
Comment (koshachy) [reply pages/docs/reference/whatsnew1430.md:424]: @sandwwraith told me that these features will be released in 1.1.0

Comment (p7nov) [reply pages/docs/reference/whatsnew1430.md]: Seems like it's 1.0.0-RC actually.

**Pull request merged**